### ISSUE TITLE
[SCREENREADER]: GIBCT® - Add aria-hidden="true" to Font Awesome icons on school and provider search results

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1462]
## Description

The Font Awesome icons are announcing themselves to screen readers. This can be improved by adding an `aria-hidden="true"` attribute to the icons per screenshot and link below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I do not want to hear the icons announced, or blank spaces announced when my virtual cursor is over the icons.
* As a visual user, I do not want the layout to change or icons to be removed from the page
* This fix should be applied to all inline icons on the GIBCT® app

## Environment

* Windows 10
* IE11, Chrome
* JAWS
* https://staging.va.gov/gi-bill-comparison-tool/profile/20020911 (example employer)
* https://staging.va.gov/gi-bill-comparison-tool/profile/15001027 (example school)

## Possible Fixes (optional)

```html
<i aria-hidden="true" class="fa fa-graduation-cap fa-search-result"></i>
```

## WCAG or Vendor Guidance (optional)

* [SO: effect of aria-hidden in fontAwesome](https://stackoverflow.com/questions/44692917/effect-of-aria-hidden-in-fontawesome/44693054#44693054)

## Screenshots or Trace Logs

![Screen Shot 2019-08-30 at 2 44 36 PM](https://user-images.githubusercontent.com/934879/64047799-6c2b6800-cb35-11e9-99d7-f1413551bd81.png)
